dir.props

```diff
@@ -96,8 +96,11 @@
         to cli.
     <RootRepo>cli</RootRepo>
     -->
-    <RootRepo>known-good</RootRepo>
-    <RootRepo Condition="$(Platform.Contains('arm'))">core-setup</RootRepo>
+
+    <!-- Temporarily update RootRepo to core-setup repo project
+    until all repos build clean out of master -->
+    <RootRepo>core-setup</RootRepo>
+    <!-- <RootRepo Condition="$(Platform.Contains('arm'))">core-setup</RootRepo> -->
```

Comment:
While it is a no-op I don't know if there is any good reason to not keep the arm condition included so we don't forget to uncomment it later.

Reply:
Fixed.